src/scripts/clipperUI/panels/dialogPanel.tsx

```diff
@@ -15,42 +17,55 @@ export interface DialogPanelProps {
 	content: HTMLElement[];
 	buttons: DialogButton[];
 	fontFamily?: Localization.FontFamily;
+	buttonFontFamily?: Localization.FontFamily;
+	divId?: string;
```

Comment:
Consider renaming this to containerId or similar, as it's hard to determine which div this refers to